### DISCUSSION TITLE
typecover on surface pro 7

### DISCUSSION THREAD

**Discussion by thornygirl** [linux-surface/linux-surface#1492] [General]
my type cover doesnt seem to wanna work. its connected via blue tooth and everything. i JUST bought this thing if it helps i have in fact downloaded the kernel and am currently running pop os

**Comment by Flameopathic** [comment]
i have had no issues; are you using a genuine type cover? if so, they don't connect via bluetooth. if it does use bluetooth and is not genuine, it seems that it would be a problem with your bluetooth rather than something related to these special kernel patches

SP7, NixOS, Hyprland
